libkbfs/backpressure_disk_limiter.go

```diff
@@ -185,6 +189,11 @@ func (bt *backpressureTracker) onBlocksDelete(blockResources int64) {
 	bt.updateSemaphoreMax()
 }
 
+func (bt *backpressureTracker) beforeDiskBlockCachePut(ctx context.Context,
```

Comment:
I agree regarding the context, but I'm going to leave the error because it allows us to explicitly check the preconditions (e.g. `blockBytes != 0`).